### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by yageek**
Conventionnally, `.go.` files' names are lowercased (=> would need to do it yourself)
- All HTTP status code are present in `net/http` package
- `const` statement can be factorized
- Use full path import package

### CODE REVIEW

File: downloader.go [hunk 1]
```diff
@@ -85,12 +101,12 @@ func (dl *downloader) Run() bool {
...
 	dl.merger.Close()
 
-	return !dl.failure
+	return nil
 }
 
 func (dl *downloader) extractFileName() {
 	tokens := strings.Split(dl.url, "/")
-	last := tokens[len(tokens) - 1]
+	last := tokens[len(tokens)-1]
```
Comment (MGaetan89) [inline downloader.go]: This change is bad, and you should feel bad

File: merger.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func (m *merger) Run() {
 func (m *merger) computeSpeed() {
 	now := time.Now().UnixNano()
 
-	if now - m.lastUpdate >= int64(time.Second) {
+	if now-m.lastUpdate >= int64(time.Second) {
```
Comment (MGaetan89) [inline merger.go]: This change is bad, and you should feel bad

**Review by MGaetan89 — CHANGES_REQUESTED** [review]

File: downloader.go (hunk 1 above)
Comment (yageek) [reply downloader.go]: Not me, `gofmt` is to blame :D

File: merger.go (hunk 2 above)
Comment (yageek) [reply merger.go]: Not me, `gofmt` is to blame :D

**Comment by MGaetan89** [comment]
Any chance to get this merge? :trollface:

**Comment by Maaarmotte** [comment]
I would like to merge it, but I cannot find the tests

**Comment by MGaetan89** [comment]
Any update on this?

**Comment by yageek** [comment]
I'll close this PR as the code seems to have been integrated in a different way

**Pull request closed without merging**